arch/vocab.md

```diff
@@ -4,32 +4,67 @@ A registry of vocabulary shared among all projects.
 # Abstract
 These concepts are not tied to any specific context.
 
-* Offer: to provide a service
+* `Offer`: to provide a service
 	- to present an option of satisfying a requirement
 	- [Spec](https://schema.org/Offer)
 	- e.g. selling tickets for an event
 
-* Section: a subdivision of a body
-	- **alias of** Subdivision
+* `Event`: an occurrence
+	- [Spec](https://schema.org/Event)
+	- e.g. a meeting
+
+* `Section`: a subdivision of a body
+	- **alias of** `Subdivision`
 	- e.g. the navy is a subdivision of the national armed forces
 
 # Academic
-* Course: 
-	- **alias of** Curricular Unit
+* `Course`: 
+	- **alias of** `Curricular Unit`
 	- [Spec](https://schema.org/Course)
 	- e.g. Software Laboratory
 
-* Class: an instance of Course in a given Semester
-	- **alias of** Course Offer
-	- **alias of** Course Instance
+* `Class`: an instance of `Course` in a given `Academic Term`
+	- **alias of** `Course Offer`
+	- **alias of** `Course Instance`
 	- [Spec](https://schema.org/CourseInstance)
-	- e.g. Software Laboratory, v1920
+	- e.g. Software Laboratory aka SL, v1920
 
-* Class Section: a subdivision of a Class
+* `Class Section`: a subdivision of a `Class`
```

Comment:
I think we should figure out a better name than `Class Section`. I really can't think of a better one though.

Should a `Class Section` be an instance of a `Class` the same way `Class` is an instance of `Course`?
> * `Class`: an instance of `Course` in a given `Academic Term`
* `Class Section`: an instance of a `Class` with a given `Schedule` and lecturer